README.md

```diff
@@ -19,6 +19,9 @@ The three main goals are:
 * miriad (aipy) -> uvfits
 * FHD -> uvfits
 * FHD -> miriad (aipy)
+* CASA measurement sets -> miriad
+* CASA measurement sets -> FHD
```

Comment:
Has a ms->pyuvdata->uvfits file been tested in FHD?  Neither I nor Bryna are aware of this having happened. Its ok if its not, we can just indicate its untested here in the doc.